dnf/logging.py

```diff
@@ -95,6 +96,17 @@ def _cfg_err_val2level(cfg_errval):
     return _ERR_VAL_MAPPING.get(cfg_errval, logging.WARNING)
 
 
+def compression_namer(name):
+    return name + ".gz"
+
+
+def compression_rotator(source, dest):
+    with open(source, "rb") as sf:
+        with gzip.open(dest, 'wb') as wf:
+            wf.write(sf.read())
```

Comment:
This will read the whole contents of the file into memory. I know the default log size is 1M, which is ok, but in case someone sets the limit much higher, I think this should read/write in chunks...

Reply:
Fixed